libraries/xcversion.rb

```diff
@@ -31,8 +31,18 @@ def list_available_xcodes
         xcversion 'list'
       end
 
+      def xcode_install_options(xcode)
+        options = ''
+
+        unless xcode.download_url.empty?
+          options = "--url '#{xcode.download_url}'"
```

Comment:
This seems too specific for the method name - I know this is the only option we're using at the moment, but that makes me think the method name should actually be `download_url_option` or something.

Reply:
good idea!